packages/features/ee/billing/credit-service.ts

```diff
@@ -550,9 +572,21 @@ export class CreditService {
 
     const billingService = new StripeBillingService();
 
-    const teamMonthlyPrice = await billingService.getPrice(process.env.STRIPE_TEAM_MONTHLY_PRICE_ID || "");
-    const pricePerSeat = teamMonthlyPrice.unit_amount ?? 0;
-    totalMonthlyCredits = (activeMembers * pricePerSeat) / 2;
+    const priceId = team.isOrganization
+      ? process.env.STRIPE_ORG_MONTHLY_PRICE_ID
+      : process.env.STRIPE_TEAM_MONTHLY_PRICE_ID;
+
+    if (!priceId) {
+      log.warn("Monthly price ID not configured", { teamId, isOrganization: team.isOrganization });
+      return 0;
+    }
+
+    const monthlyPrice = await billingService.getPrice(priceId || "");
+    const pricePerSeat = monthlyPrice.unit_amount ?? 0;
+
+    // Teams get 50% of the price as credits, organizations get 20%
+    const creditMultiplier = team.isOrganization ? 0.2 : 0.5;
+    totalMonthlyCredits = activeMembers * pricePerSeat * creditMultiplier;
```

Comment:
Why is this change needed with this PR? Is it broken on production for SMS credits as we seem to introduce ORG_MONTHLY_PRICE_ID here

Reply:
Also these magic numbers of 0.2 and 0.5 should either be in constants or configurable via env variable if it makes sense

Reply:
These changes were added by Carina here https://github.com/calcom/cal.com/pull/23077/files